src/validator.cpp

```diff
@@ -1434,7 +1434,7 @@ void updateBaseUnitCount(const ModelPtr &model,
                 }
             }
         } else if (unitMap.find(uName) == unitMap.end()) {
-            unitMap.emplace(std::pair<std::string, double>(uName, direction * uExp));
+            unitMap[uName] = direction * uExp;
```

Comment:
Can't understand why the `std::pair` was created here but as above I would prefer the `emplace` variant.

Reply:
Done in commit 50dc8441.